Library/Formula/corectl.rb

```diff
@@ -0,0 +1,37 @@
+require "language/go"
+
+class Corectl < Formula
+  desc "CoreOS over OS X made very simple"
+  homepage "https://github.com/TheNewNormal/corectl"
+  url "https://github.com/TheNewNormal/corectl/archive/v0.0.18.tar.gz"
+  sha256 "8646c18ec0414a8f6e6330bc6720d9f0c4356ae7168b18cbc371220fe7a3e087"
+  head "https://github.com/TheNewNormal/corectl.git", :branch => "golang"
+
+  depends_on "go" => :build
+  depends_on "godep" => :build
+  depends_on "gnu-sed" => [:build, "with-default-names"]
+  depends_on :macos => :yosemite
+
+  def install
+    ENV["PATH"] = ENV["PATH"].sub("#{Superenv.bin}:", "")
+    ENV["GOPATH"] = buildpath
+
+    mkdir_p buildpath/"src/github.com/TheNewNormal/"
+    ln_s buildpath, buildpath/"src/github.com/TheNewNormal/#{name}"
+    Language::Go.stage_deps resources, buildpath/"src"
+
+    args = []
+    args << "VERSION=#{version}" if build.stable?
+    system "make", "clean"
+    system "make", "corectl", *args
+    system "make", "documentation/man"
+
+    bin.install "corectl"
+    man1.install Dir["documentation/man/*.1"]
+    share.install Dir["cloud-init", "profiles"]
+  end
+
+  test do
+    system "#{bin}/corectl", "version"
```

Comment:
made test a little bit more substantive but given the nature of this (and until i get a proper test suite for it) not too simple to do substantive testing in the formula (this is an hypervisor after all)